src/python-common/ceph/deployment/drive_selection/selector.py

```diff
@@ -131,8 +131,12 @@ def assign_devices(self, device_filter):
         for disk in self.disks:
             logger.debug("Processing disk {}".format(disk.path))
 
+            if disk.being_replaced:
+                logger.debug('Ignoring disk {} as it is being replaced.'.format(disk.path))
+                continue
+
             if not disk.available and not disk.ceph_device:
-                logger.debug(
+                logger.error(
```

Comment:
does this actually need to be logged at error level? I think devices that meet this criteria could just stay in that state and we could end up doing error level logs repeatedly for the same device still being unavailable.

Reply:
this is a legacy from a debug session